### ISSUE TITLE
Enhancing Predictive Calc with Additional Models and Hyperparameter Tuning

### ISSUE THREAD

**Issue by murtaza-sadri-19** [yashasvini121/predictive-calc#8]
To further improve the capabilities of Predictive Calc under **GSSoC'24 Extd**, I propose adding new **machine learning algorithms** in form of projects and implementing **hyperparameter tuning** techniques. By incorporating a wider range of models and optimizing their parameters, we can enhance the accuracy, flexibility, and overall performance of the tool. This will provide users with more options to choose the best model for their specific prediction tasks and ensure that the models are fine-tuned to achieve optimal results.

I wish to contribute this to your repository. Kindly assign me this.

**Comment by yashasvini121** [comment]
Hi @murtaza-sadri-19, for now, please be more specific about your proposal.
Describe one problem you plan to tackle and the model you'd create for it. For example, the project currently includes a House Price Prediction model using regression. You could develop multiple models or document both successful and rejected models in a Jupyter notebook. Also, mention the estimated time that you will require.

Make sure to review the [add new model template](https://github.com/yashasvini121/predictive-calc/issues/new?assignees=&labels=&projects=&template=add-new-model.md&title=) to follow the proper issue format.

**Comment by murtaza-sadri-19** [comment]
Like I'm Planning to implement Hyperparameter tuning in a model which would classify **Breast cancer** tumor into two classes _viz._ **Benign/Malignant**. And I would also apply ensemble learning techniques over the dataset. 

I would require approx 3-4 days to complete the same. And regarding the format I would make that I comply to that.

**Comment by yashasvini121** [comment]
Ok, great, go ahead!

**Comment by yashasvini121** [comment]
Updates @murtaza-sadri-19?

**Comment by murtaza-sadri-19** [comment]
> Updates @murtaza-sadri-19?

**Comment by yashasvini121** [comment]
Closing this issue as a similar one has been created under #33 by you.
